app/views/content/train-to-be-a-teacher/if-you-want-to-change-career.md

```diff
@@ -8,7 +8,7 @@ related_content:
     Changing career from engineering to teaching physics: "/blog/natural-transition-from-engineering-to-teaching-physics"
     Engineers teach physics teacher training courses : "/subjects/engineers-teach-physics"
     Transferring my skills to teaching: "/blog/transferring-my-skills-to-teaching"
-    Initial teacher training : "/train-to-be-a-teacher/initial-teacher-training"
+    Fund your training if you're a parent or carer: "/funding-and-support/if-youre-a-parent-or-carer"
```

Comment:
@gemmadallmandfe @philspray not sure the related content section is necessarily the best place for this link, but given the career changers page is not fantastic atm and is on our backlog to look at any way, thought I'd just pop it in there for now